### ISSUE TITLE
[HOLD for payment 2023-05-02] [$1000]  Inconsistent capitalization in product

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#17239]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Go to web chrome
2. Go to any chat
3. Send message and hover over the 'Copy to clipboard' and 'Mark as unread'.
4. Notice that only the first letter is in the capital form
5. But now hover over the call icon and notice that the first letter of the Call is in the capital. (Even in web messenger, it's 'Start a video call', where 'c' is small)
-----
1. Go to the Settings > Payments
2. Select a payment method to delete
-----
1. Open Chat
2. Upload a attachment
-----
1. Open the app
2. Open settings
3. Open workspaces
4. Open any workspace
5. Open connect bank account
6. Click on connect manually
7. Without filling any details, click on continue and observe that Routing number and Account number fields as 'N' small in labels but capital in error
8. Fill in details in step 1 and click on continue. eg details: routing no= 210021002, acc no= 1234
9. On step 2, check below company address field, instruction text has 'b' small for boxes.
10. Without filling any field, click on Save & continue
11. Observe that for company address field, validation message has 'b' capital for box
12. Observe that for fields like phone number, zip code, there is no capital word in validation
13. Observe that 'Tax ID number' field has small 'n' in label and capital 'N' in validation error
14. Observe that 'Company type' field has small 't' in label and capital 'T' in validation error
15. Observe that for 'Incorporation state' field, even though field is dropdown, validation error says 'Please enter' in place of 'Please select'

## Expected Result: 

- To maintain consistency between the other tooltips, the 'C' should be small in the tooltip for the call icon
-----
- The header in the Delete Account modal "A" should be in small letters, as the header in the "Delete workspace" modal "w" is in small letters and same in "Delete comment" modal
-----
- Uploading Attachment... message "A" should be in small letters
-----
- App should maintain consistent capitilization and validation statements for fields in connect bank account

## Actual Result: 

- The 'C' is in the capital letter in the call icon tooltip
-----
- The header "Delete Account" in the Delete Account modal "A" is capitalized
-----
- Uploading Attachment... message "A" is capitalized
-----
- App has many inconsistencies in capitilization and validation statements for fields in connect bank account

## Workaround:
N/A

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:**  1.2.98-2
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/43996225/230997926-518af286-0318-4f13-a3ad-879f7a4fd7ab.mp4

<img width="960" alt="Untitled" src="https://user-images.githubusercontent.com/43996225/230997953-c3329208-b9e1-4629-98e2-4e8a464ec4c2.png">

**Expensify/Expensify Issue URL:**
**Issue reported by:** @ayazhussain79
**Slack conversation:** [expensify.slack.com/archives/C049HHMV9SM/p1680972128646799](https://expensify.slack.com/archives/C049HHMV9SM/p1680972128646799)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01ecbf592ef39ed301</li>
        <li>Upwork Job ID: 1645820257278033920</li>
        <li>Last Price Increase: 2023-04-11</li>
    </ul>
</details>

**Comment by MelvinBot** [comment]
Triggered auto assignment to @sakluger (`Bug`), see https://stackoverflow.com/c/expensify/questions/14418 for more details.

**Comment by sakluger** [comment]
@Expensify/design should we be following the same capitalization scheme for the `Start a call` hover text as we use for other hover texts? If so, I think I can easily change this myself [in the code here](https://github.com/Expensify/App/blob/67097aa547de76f86c91b6405d8c902e688c8159/src/languages/en.js#L110).

**Comment by shawnborton** [comment]
I agree that we should be using sentence case everywhere, not title case. That's probably more of a marketing question but I think they would answer the same way.

**Comment by sakluger** [comment]
Okay cool, I'll add `external` to get this moving. Should be quick and easy!

**Comment by MelvinBot** [comment]
Current assignee @sakluger is eligible for the External assigner, not assigning anyone new.

**Comment by MelvinBot** [comment]
Triggered auto assignment to Contributor-plus team member for initial proposal review - @0xmiroslav (`External`)

**Comment by dukenv0307** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.

Capitalization of three-word tooltips is not consistent.

### What is the root cause of that problem?

The literal defined in language file is not as expected:

https://github.com/Expensify/App/blob/67097aa547de76f86c91b6405d8c902e688c8159/src/languages/en.js#L110

### What changes do you think we should make in order to solve the problem?

Modify the literal in the above mentioned line as expected: `Make a call`.

I think if we agree upon this capitalization format (i.e. Sentence-style `Make a call` not Title-style `Make a Call`), we should inspect the whole language file and modify where necessary.

Some of the same issues: https://github.com/Expensify/App/issues/17201, https://github.com/Expensify/App/issues/17243.

### What alternative solutions did you explore? (Optional)

NA

**Comment by MelvinBot** [comment]
Triggered auto assignment to @thienlnam (`External`), see https://stackoverflow.com/c/expensify/questions/7972 for more details.

**Comment by PrashantMangukiya** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
The tooltip has capital 'C' in the word Call unlike other tooltips with three words

### What is the root cause of that problem?
Start a call button tooltip rendered via this line
https://github.com/Expensify/App/blob/5df7225e347253fdb537d282585774dc6cb4c040/src/components/VideoChatButtonAndMenu/BaseVideoChatButtonAndMenu.js#L105

Within translation file it consist this value
https://github.com/Expensify/App/blob/5df7225e347253fdb537d282585774dc6cb4c040/src/languages/en.js#L197-L199

We can see at line 197 there is a `C` in capital for `Start a Call`. This is the root cause of the problem.

### What changes do you think we should make in order to solve the problem?
We have to make c small as shown below:
```javascript
 tooltip: 'Start a call', 
```

So this will solve the problem.


Note: I will also suggest to change translation key from `tooltip` to `startACall` so key remain consistence with other button keys like `zoom` and `googleMeet` etc.

### What alternative solutions did you explore? (Optional)
None

**Comment by neonbhai** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Inconsistent capitalization of the ```'Start a Call'``` tooltip, unlike other three word tooltips.

![start a call ](https://user-images.githubusercontent.com/64629613/231243367-a4bd0fdd-2453-4f90-b3a3-60bfef173a48.png) 
^inconsistent with:
![clipboard](https://user-images.githubusercontent.com/64629613/231243916-16f27caf-02e6-4520-ad8d-f73792a4d004.png) ![mark as unread](https://user-images.githubusercontent.com/64629613/231243480-fc183ef0-c299-4ba8-b98f-ca45718206b7.png)



### What is the root cause of that problem?
The tooltip references this string which is not defined correctly the en.js file.
https://github.com/Expensify/App/blob/67097aa547de76f86c91b6405d8c902e688c8159/src/languages/en.js#L110

### What changes do you think we should make in order to solve the problem?
We should change the String for the tooltip to ```'Start a call'```

### What alternative solutions did you explore? (Optional)
NA


(Hi, first time contributor here, I like React Native and have been wanting to contribute but I am intimidated with the complexity of the codebase. Since this is a simple change, Can I be assigned this as submitting this PR will help build my confidence.)

**Comment by neonbhai** [comment]
Contributor details
Your Expensify account email: someshwartripathi8@gmail.com
Upwork Profile Link: https://www.upwork.com/freelancers/~01b1623e7bd66df3df

**Comment by MelvinBot** [comment]
✅ Contributor details stored successfully. Thank you for contributing to Expensify!

**Comment by PrashantMangukiya** [comment]
@0xmiroslav  First proposal suggested to Change it `Make a call` actually it should be `'Start a call'. I think what I suggested https://github.com/Expensify/App/issues/17239#issuecomment-1503682900 is correct grammar to change. 

I also suggested  to change translation key from tooltip to `startACall` so key remain consistence with other button keys like zoom and googleMeet etc.

**Comment by PrashantMangukiya** [comment]
@0xmiroslav For such small correction such grammar matters so I requested to consider it.

**Comment by thienlnam** [comment]
Great catch @0xmiroslav, I agree we should just take care of all mis-capitalization in this issue - I've commented on those other issues and so can we get another proposal here to handle all these capitalization differences in our language files?

**Comment by thienlnam** [comment]
Updated solution should check en.js and es.js and make sure we're following the pattern of just the first word capitalized

**Comment by priya-zha** [comment]
@thienlnam Wanted to ask, will I not be considered for the issue related to the 'Start a Call' bug?

**Comment by thienlnam** [comment]
@priya-zha The bug you reported was for incorrect capitalization, which was already reported a few days ago [here](https://expensify.slack.com/archives/C049HHMV9SM/p1680972128646799).  It was for a different copy, but since it basically handles the same issue (incorrect capitalization) it is considered a duplicate and won't be eligible for a reporting bonus

**Comment by MelvinBot** [comment]
📣 @dukenv0307 You have been assigned to this job by @thienlnam!
Please apply to this job in Upwork and leave a comment on the Github issue letting us know when we can expect a PR to be ready for review 🧑‍💻
Keep in mind: [Code of Conduct](https://github.com/Expensify/App/blob/main/CODE_OF_CONDUCT.md) | [Contributing](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md) 📖

**Comment by dukenv0307** [comment]
@0xmiroslav I just updated [my proposal](https://github.com/Expensify/App/issues/17239#issuecomment-1503682816) with some similar issues. Please help to check and confirm which one we should fix
cc @thienlnam

**Comment by sakluger** [comment]
Great idea to review similar issues in the language file. Let's update numbers 1, 3, 4, 6, and 12.

Going through the reasoning for each proposed change:

1. Transfer balance - Yes, let’s use sentence case for both the link and the header
2. Tap Settings - No, “Settings” is the button name so it should be capitalized
3. Terms of Service - Yes, we should standardize to `Terms of Service`
4. Choose account - Yes, use sentence case
5. Expense Management/Spend Management/Expense Reports/Company Credit Card/REceipt Scanning App/Bill Pay - No, these are in the footer of the sign in page and should have Title case
6. Delete account - Yes, use sentence case
7. CPA Card - No, footer copy should be Title case
8. Press Kit - No, footer copy
9. No, footer copy
10. No, footer copy
11. Your Expensify Cards are ready to go - no, `Expensify Cards` is a product name
12. Uploading attachment - Yes, use sentence case

**Comment by dukenv0307** [comment]
@0xmiroslav @thienlnam [The PR](https://github.com/Expensify/App/pull/17327) is ready for review

**Comment by thienlnam** [comment]
@sakluger Is a better contact for confirming these copies - if the PR is delayed due to us the bonus amount will still apply so no worries there

**Comment by sakluger** [comment]
Sorry for the delay. Yes, let's do sentence case for "Debit Card", "Business Days" and "Bank Account". So `Debit card`, `Business days`, `Bank account`.

**Comment by sakluger** [comment]
By the way, since the delay here was my fault, this will still qualify for the efficiency bonus as long as we merge by tomorrow.

**Comment by sakluger** [comment]
By the way, since the delay here was my fault, this will still qualify for the efficiency bonus as long as we merge by tomorrow.

**Comment by thienlnam** [comment]
Yeah I agree, going to notify Kavi to consolidate the duplicate issues for capitalization in here

**Comment by dhanashree-sawant** [comment]
Hi @0xmiroslav, @thienlnam, coming from the 17539 issue, if it is also solved along with this issue, will I be eligible for reporting bonus?

**Comment by thienlnam** [comment]
Sorry, same applies here since this is a duplicate
https://github.com/Expensify/App/issues/17239#issuecomment-1503880492

**Comment by dhanashree-sawant** [comment]
Hi @thienlnam , my point here is although the bug is about incorrect capitalization, 17239 or any discussion till now didn't focused on connect bank account validation capitalization. If not for 17539 report, we would have not worked on it and it may have again come up in future as new issue which would have been worked separately.

Also one small part different from capitalization in the report was step number 15 in action performed. It is about wrong terminology.

**Comment by thienlnam** [comment]
Yes, though that would apply to any part of the product that has inconsistent capitalization and any of the previous issues that mentioned incorrect capitalization. 

Since there were so many different reports we consolidated the issues addressing inconsistent capitalization and are addressing all of them from our [constants file](https://github.com/Expensify/App/issues/17239#issuecomment-1503877345). All these are providing different examples, but if some are missed that would be a regression from this issue which aims to resolve all 'inconsistent capitalization in product'

**Comment by dhanashree-sawant** [comment]
I appreciate the initiative of working all these issues together, it will rectify many issues at once but am not sure if its fair to every reporter as they have contributed in it but won't be rewarded only because all of them are worked under one PR. Can you also let me know if point 15 of action's performed in #17539 will also be taken care in this issue? It is not about capitalization.

**Comment by thienlnam** [comment]
> Can you also let me know if point 15 of action's performed in https://github.com/Expensify/App/issues/17539 will also be taken care in this issue? It is not about capitalization.

Thanks for pointing this out - I actually reopened your issue because it is not related to capitalization and cleaned up the issue there.

> I appreciate the initiative of working all these issues together, it will rectify many issues at once but am not sure if its fair to every reporter as they have contributed in it but won't be rewarded only because all of them are worked under one PR.

As for this, I understand the concern about fairness - I'm consolidating them because the underlying issue for all of these is incorrect capitalization which can be resolved by just looking at the constants file where all of our strings exist and so here we're just giving the bug report to the first person that reported the problem.

For example, if there is a bug in a component which is used in multiple locations - there are typically multiple bug reports for a problem in different parts of the app. In this case, we'll resolve the bug for the first issue and then we just close the rest of the bug reports because they were duplicates.

**Comment by dukenv0307** [comment]
@0xmiroslav with "Tax ID Number", "PO Box" it should be all in normal-case or "ID", "PO" is still in capital

**Comment by dukenv0307** [comment]
@sakluger ping again in case you miss it, please help to check [this comment](https://github.com/Expensify/App/issues/17239#issuecomment-1514107915)

**Comment by sakluger** [comment]
Yes "P" should be capitalized.

**Comment by MelvinBot** [comment]
`Reviewing` label has been removed, please complete the "BugZero Checklist".

**Comment by MelvinBot** [comment]
The solution for this issue has been :rocket: deployed to production :rocket: in version 1.3.4-0 and is now subject to a 7-day regression period :calendar:. Here is the list of pull requests that resolve this issue:

- https://github.com/Expensify/App/pull/17327


If no regressions arise, payment will be issued on 2023-05-02. :confetti_ball:

After the hold period is over and BZ checklist items are completed, please complete any of the applicable payments for this issue, and check them off once done.
- [ ] External issue reporter
- [ ] Contributor that fixed the issue
- [ ] Contributor+ that helped on the issue and/or PR

As a reminder, here are the bonuses/penalties that should be applied for any External issue:
- Merged PR within 3 business days of assignment - 50% **bonus**
- Merged PR more than 9 business days after assignment - 50% **penalty**

**Comment by MelvinBot** [comment]
**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

- [ ] [@0xmiroslav] The PR that introduced the bug has been identified. Link to the PR:
- [ ] [@0xmiroslav] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
- [ ] [@0xmiroslav] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
- [ ] [@sakluger] Determine if we should create a regression test for this bug.
- [ ] [@0xmiroslav] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
- [ ] [@sakluger] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

**Comment by sakluger** [comment]
Just noticed this one never switched back to Daily and I missed it, sorry about that! I'm reviewing now to see what is outstsanding before paying out.

**Comment by sakluger** [comment]
Based on the timeline including times waiting for copy, I think this should qualify for the efficiency bonus.

@ayazhussain79 @dukenv0307 @0xmiroslav I've sent you all offers in Upwork.

**Comment by ayazhussain79** [comment]
@sakluger offer accepted

**Comment by sakluger** [comment]
All paid.
